web/_js/config.js

```diff
@@ -428,6 +428,11 @@ const externalLinksConfig = [
 	},
 ];
 
+const disclaimerConfig = {
+	showWarning: true
+};
+window.disclaimerConfig = disclaimerConfig;
+
```

Comment:
Again, move this to `main.js` because this file is for instance configuration.

Reply:
Moved